pkg/volume/empty_dir/empty_dir_quota.go

```diff
@@ -0,0 +1,66 @@
+package empty_dir
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/resource"
+	"k8s.io/kubernetes/pkg/volume"
+)
+
+// EmptyDirQuotaPluginBuilder is a simple wrapper for the k8s empty dir plugin builder.
+type EmptyDirQuotaPluginBuilder struct {
+	volume.VolumePlugin
```

Comment:
inlining upstream types is a good way to silently break when an interface changes and the functions you thought you were overriding no longer get called